docs/reference/aggregations/metrics/scripted-metric-aggregation.asciidoc

```diff
@@ -9,21 +9,19 @@ Example:
 
 [source,js]
 --------------------------------------------------
-POST ledger/_search?size=0
+POST /ledger/_search
 {
-    "query" : {
-        "match_all" : {}
-    },
-    "aggs": {
-        "profit": {
-            "scripted_metric": {
-                "init_script" : "params._agg.transactions = []",
-                "map_script" : "params._agg.transactions.add(doc.type.value == 'sale' ? doc.amount.value : -1 * doc.amount.value)", <1>
-                "combine_script" : "double profit = 0; for (t in params._agg.transactions) { profit += t } return profit",
-                "reduce_script" : "double profit = 0; for (a in params._aggs) { profit += a } return profit"
-            }
-        }
+  "size": 0,
+  "aggs": {
+    "profit": {
+      "scripted_metric": {
+        "init_script" : "params._agg.transactions = []",
+        "map_script" : "params._agg.transactions.add(doc['type.keyword'].value == 'sale' ? doc.amount.value : -1 * doc.amount.value)", <1>
```

Comment:
makes sense. I was adding the `.keyword` as the default behavior from elasticsearch when no explicit mappings were provided. This page didn't have mappings for the examples (but the test does), so I assumed this was a miss.